data/tutorials/getting-started/gs_07_operators.md

```diff
@@ -0,0 +1,196 @@
+---
+id: operators
+title: Operators
+descrption: |
+  
+category: "Getting Started"
+---
+
+# Operators
+
+## Using Binary Operators
+
+In OCaml almost all binary operators are regular functions. The function underlying behind an operator is referred by surrounding the operator symbol with parenthesis. Here are the addition, string concatenation and equality functions:
+
+```ocaml
+# (+);;
+- : int -> int -> int = <fun>
+# (^);;
+- : string -> string -> string = <fun>
+# (=);;
+- : 'a -> 'a -> bool = <fun>
+```
+
+Note the operator symbol for multiplication is ` * `, but OCaml comments also have an opening delimiter `(*` and a closing delimiter `*)`. To resolve the parsing ambiguity, space characters must be inserted to get the multiplication function.
+
+```ocaml
+# ( * );;
+- : int -> int -> int = <fun>
+```
+
+Using operators as functions is convenient when combined with partial application. For instance, here is how to get the values which are greater or equal to 10 in a list of integers, using the function `List.filter` applied to the function `( <= ) 10`:
+```ocaml
+# List.filter;;
+- : ('a -> bool) -> 'a list -> 'a list = <fun>
+
+# List.filter (( <= ) 10);;
+- : int list -> int list = <fun>
+
+# List.filter (( <= ) 10) [6; 15; 7; 14; 8; 13; 9; 12; 10; 11];;
+- : int list = [15; 14; 13; 12; 10; 11]
+```
+
+The first two lines are only informative.
+
+1. The first shows the `List.filter` type, which is a function taking two parameters. The first parameter is a function; the second is a list.
+2. The second is the partial application of `List.filter` to `( <= ) 10`, which is a function returning `true` if applied to a number which is greater or equal than 10.
+
+Finally, in the third line, all the arguments expected by `List.filter` are provided. The returned list contains the values satisfying the `( <= ) 10` function.
+
+## Defining Binary Operators
+
+It is also possible to define binary operators. Here is an example:
+```ocaml
+# let cat s1 s2 = s1 ^ " " ^ s2;;
+val cat : string -> string -> string = <fun>
+
+# let ( ^? ) = cat;;
+val ( ^? ) : string -> string -> string = <fun>
+# "hi" ^? "friend";;
+- : string = "hi friend"
+```
+
+## Unary Operators
+
+Unary operators are also called prefix operators. In some contexts, it can make sense to shorten the name of a function into a symbol. This is often used as a way to shorten the name of a function which performs some sort of conversion over its argument.
```

Comment:
```suggestion
Unary operators are also called prefix operators. In some contexts, it can make sense to shorten a function's name into a symbol. This is often used as a way to shorten the name of a function which performs some sort of conversion over its argument.
```

This last sentence it's clear to me. Does shortening the name perform some sort or conversion or does the name of the function itself?

Reply:
It is the function which does a conversion of some sort